views/settings/fields/rocket-cdn.php

```diff
@@ -35,6 +35,16 @@
 					<?php endif; ?>
 				</div>
 			<?php endforeach; ?>
+		<?php else : ?>
+			<div class="wpr-text">
+				<input type="text" name="wp_rocket_settings[cdn_cnames][]" value="" placeholder="xxxxxx.rocketcdn.me" />
```

Comment:
Yeah that's because of the way the UI was made orginally. I could add one and hide it with the `screen-reader-text` class, what do you think?

Reply:
Yes. Moreover, since `placeholder` attributes are not read by screen readers (as far as I know, but maybe it has changed), the users have no way to guess what this field is for.
Do you think we have more cases like this in the plugin? Maybe it could be a good idea to plan a general a11y checkup at some point, especially with all the JS we pile up.

Reply:
We probably do, I have no idea how accessible the settings page is. I already suggested an a11y audit, I don't know if and when it will happen, if ever.

Reply:
Currently our UI is not a11y compliant. 

In a later task, we will do an a11y audit and then bring our UI into compliance.